src/utils.py

```diff
@@ -0,0 +1,135 @@
+from __future__ import print_function
+
+import os
+import io
+import sys
+import imp
+import json
+from os.path import join, exists
+from collections import OrderedDict
+
+ENCODING = 'ISO-8859-1'
```

Comment:
This should be in one place in the source code. hopefully we don't have to put it in several places. It's fine having several of these initialization in test modules

Reply:
It's only present in the `utils` file. Sorry if I didn't understand you.